roles/atomic_upgrade/tasks/main.yml

```diff
@@ -34,10 +34,9 @@
 
 - block:
 
-    - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+    - name: Workaround old atomic command bug: failure b/c /tmp is a symlink
```

Comment:
TBH, I dunno if that's even the same error we see on RHEL.  Having a misleading comment is more unhelpful than having no comment.  Pretty sure it does say "/tmp has too many levels of symlinks".  You think it should be like:

``Workaround fsetxattr bug in older atomic command versions``